internal/apps/ai-proxy/providers/metrics/collector.go

```diff
@@ -82,14 +84,37 @@ type lvsDistincter struct {
 }
 
 var mockedAudit = models.AIProxyFilterAudit{
```

Comment:
maybe `mockedAudit` should be put at `_test.go` files.

Reply:
removed it